[2025-02-28T04:35:04.950+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T04:30:00+00:00 [queued]>
[2025-02-28T04:35:04.960+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T04:30:00+00:00 [queued]>
[2025-02-28T04:35:04.960+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-02-28T04:35:04.973+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-02-28 04:30:00+00:00
[2025-02-28T04:35:04.979+0000] {standard_task_runner.py:60} INFO - Started process 1527 to run task
[2025-02-28T04:35:04.982+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'scheduled__2025-02-28T04:30:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpxndubagp']
[2025-02-28T04:35:04.984+0000] {standard_task_runner.py:88} INFO - Job 36: Subtask load_to_postgres_file
[2025-02-28T04:35:05.028+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T04:30:00+00:00 [running]> on host 335ace54b661
[2025-02-28T04:35:05.095+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T04:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T04:30:00+00:00'
[2025-02-28T04:35:05.098+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-02-28T04:35:05.099+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-02-28T04:35:05.109+0000] {subprocess.py:86} INFO - Output:
[2025-02-28T04:35:05.938+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/02/28/11-35-03.csv' từ MinIO
[2025-02-28T04:35:05.939+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-02-28T04:35:05.939+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 1 file mới từ MinIO.
[2025-02-28T04:35:06.010+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-28T04:35:06.037+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250228T043000, start_date=20250228T043504, end_date=20250228T043506
[2025-02-28T04:35:06.080+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-28T04:35:06.110+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
